### DISCUSSION TITLE
I have managed to migrate the pipelines to GitLab - Would like to share

### DISCUSSION THREAD

**Discussion by powershellwhizz** [Azure/AzOps#503] [Ideas]
I am new to all of this. Never really got into DevOps and IaC until last month when I was tasked with getting AzOps to work in our GitLab environment. It was a nightmare but I finally finished it a fe days ago. I would like to share this with the community but I have no idea what the process is or whether I would potentially breach any licensing agreements. I guess I can upload my version of AzOps with the GitLab pipelines and make a pull request? I was a little concerned however, that since I am new to this, if my code needs to meet some level of quality I am not aware of? Any advice from the AzOps team? I know it would benefit a lot of people who have on-prem GitLab.

thanks,

**Comment by daltondhcp** [comment]
Hey @powershellwhizz,

That sounds absolutely amazing! Since we currently 'officially' only support Github Actions and ADO, my recommendation would be that you first publish your artifacts/docs in your private "AzOps-GitLab" GitHub repository to get it out there. Then we can review and decide if we want expand our official support and if and how to publish of that into our AzOps-Accelerator starter repository.

Let me know if you need any assistance!

/Johan

**Comment by daltondhcp** [comment]
Hey @powershellwhizz, did you ever push through with this?

**Comment by powershellwhizz** [comment]
Hi dalton,
I finished it but I never published it as I never got round to it and didn't think it would be included to AzOps. Is this something you are now interested in, or are you just chasing down old tickets?

**Reply by daltondhcp** [reply]
Hey,

A little bit of both. 
Please reach out to me at jodahlbo[at]microsoft.com to discuss what we can do to get it published. 

Thanks,
Johan
